BDArmory/Weapons/Missiles/MissileLauncher.cs

```diff
@@ -1189,6 +1196,7 @@ public override void OnFixedUpdate()
                     {
                         Detonate();
                     }
+                    if (Vector3.Distance(transform.position, launchPosition) > engageRangeMax) Detonate();
```

Comment:
Doesn't this ignore the extra distance from the max DLZ due to launching at high speed?